setup.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+"""
+setup.py
+A module that installs agol-validator as a module
+"""
+from glob import glob
+from os.path import basename, splitext
+
+from setuptools import find_packages, setup
+
+setup(
+    name='agol-validator',
+    version='0.9.0',
+    license='MIT',
+    description='Validates all hosted feature service items in a user\'s AGOL folders',
+    author='Jake Adams',
+    author_email='jdadams@utah.gov',
+    url='https://github.com/agrc/agol-validator',
+    packages=find_packages('src'),
+    package_dir={'': 'src'},
+    py_modules=[splitext(basename(path))[0] for path in glob('src/*.py')],
+    include_package_data=True,
+    zip_safe=True,
+    classifiers=[
+        # complete classifier list: http://pypi.python.org/pypi?%3Aaction=list_classifiers
+        'Development Status :: 4 - Beta',
+        'Intended Audience :: System Administrators',
+        'Topic :: Utilities',
+    ],
+    project_urls={
+        'Issue Tracker': 'https://github.com/agrc/agol-validator/issues',
+    },
+    keywords=['gis'],
+    install_requires=[
```

Comment:
**issue**: you are missing arcgis and pandas here from my quick check

Reply:
good check; arcpy also.

This brings up a question. You don't just `pip\conda install arcpy`, it's either in your environment from an ArcGIS install or its not. Does adding something to `install_requires` that isn't in pip cause the install to fail? Is this the best way to handle an external dependency like arcpy?

Reply:
arcpy is not pip installable so i would not add it to the list or I believe it will fail the install.